samples/extensions/color_write_enable/color_write_enable.h

```diff
@@ -0,0 +1,56 @@
+/* Copyright (c) 2023, Mobica Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "api_vulkan_sample.h"
+
+/**
+ * @brief Color generation toggle using VK_EXT_color_write_enable
+ */
+class ColorWriteEnable : public ApiVulkanSample
+{
+  public:
+	bool  r_bit_enabled      = true;
```

Comment:
Shouldn't these values be private?

Reply:
They definitely should be. I have fixed them.